src/disco/css/App.scss

```diff
@@ -43,6 +51,25 @@ header {
   }
 }
 
+.cta {
+  align-self: center;
+  padding: 1em 1.5em;
+  margin: 30px 0;
+  background: $cta-background-color;
+  color: #fff;
+  text-decoration: none;
+  font-size: 13px;
+  transition: background .2s;
```

Comment:
Yep I like the idea of ms for this. I'll change that and we should add that to the style guide when we get around to writing one :)